packages/utilities/psammead-styles/README.md

```diff
@@ -1,6 +1,11 @@
 # psammead-styles &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-styles.svg)](https://www.npmjs.com/package/@bbc/psammead-styles) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/BBC-News/psammead/blob/latest/CONTRIBUTING.md)
 
-This package is a collection of variable constants which can be imported into your application.
+This package provides a collection of String constants for use in CSS, containing non-GEL styling details that are bespoke to specific services.
+
+## Exports
+
+`/colours` - Project-defined colours that will be required by multiple Psammead components or themes. These colours are not defined by GEL.
+`/fonts` - Project-defined font families. These fallbacks are not defined by GEL.
```

Comment:
Are these fallbacks?

Reply:
The font fallbacks -- i.e. "Reith, then Helvetica". I was trying to communicate that these are specific to Articles, hence their inclusion in this package, & not `gel-foundations`. Any suggestions? 🤔